deploy/complete/terraform/mushop.tf

```diff
@@ -7,7 +7,7 @@ resource "kubernetes_namespace" "mushop_namespace" {
   metadata {
     name = "mushop"
   }
-  depends_on = [oci_containerengine_node_pool.oke-mushop_node_pool]
+  depends_on = [oci_containerengine_node_pool.oke_mushop_node_pool]
```

Comment:
does the provider wait for the node to be in the AVAILABLE state ?

Reply:
No, and is not needed. If the cluster is available, everything get scheduled to provision as soon is available.

oke terraform node_pool resource does not have any status control. This does not affects the solution deployment, as measures have been applied to have everything gracefully.